usecases/backup/coordinator.go

```diff
@@ -188,30 +189,38 @@ func (c *coordinator) Restore(ctx context.Context, req *backup.DistributedBackup
 
 	go func() {
 		c.commit(context.Background(), &statusReq, nodes)
+		if err := store.PutMeta(ctx, GlobalRestoreFile, &c.descriptor); err != nil {
+			c.log.WithField("action", OpCreate).
+				WithField("backup_id", req.ID).Errorf("put_meta: %v", err)
+		}
 	}()
 
 	return nil
 }
 
-func (c *coordinator) OnStatus(ctx context.Context, store coordStore, req *StatusRequest) (reqStat, error) {
+func (c *coordinator) OnStatus(ctx context.Context, store coordStore, req *StatusRequest) (*Status, error) {
 	// check if backup is still active
 	st := c.lastOp.get()
 	if st.ID == req.ID {
-		return st, nil
+		return &Status{Path: st.Path, StartedAt: st.Starttime, Status: st.Status}, nil
+	}
+	filename := GlobalBackupFile
+	if req.Method == OpRestore {
+		filename = GlobalRestoreFile
 	}
-
 	// The backup might have been already created.
-	meta, err := store.Meta(ctx, req.ID)
+	meta, err := store.Meta(ctx, filename)
 	if err != nil {
 		path := fmt.Sprintf("%s/%s", req.ID, GlobalBackupFile)
-		return reqStat{}, fmt.Errorf("%w: %q: %v", errMetaNotFound, path, err)
+		return nil, fmt.Errorf("%w: %q: %v", errMetaNotFound, path, err)
```

Comment:
Should the last `%v` should be a `%w`?

Reply:
We can only use one %w in an expression.
The first one is the most important one